mapper/mapper.go

```diff
@@ -12,7 +15,8 @@ import (
 )
 
 // CreateSearchPage maps type searchC.Response to model.Page
-func CreateSearchPage(cfg *config.Config, req *http.Request, basePage coreModel.Page, validatedQueryParams data.SearchURLParams, categories []data.Category, topicCategories []data.TopicCategory, respC searchC.Response, departments searchC.Department, lang string, ErrorMessage string) model.SearchPage {
+
+func CreateSearchPage(cfg *config.Config, req *http.Request, basePage coreModel.Page, validatedQueryParams data.SearchURLParams, categories []data.Category, topicCategories []data.TopicCategory, respC searchC.Response, departments searchC.Department, lang string, homepageResponse zebedee.HomepageContent, ErrorMessage string) model.SearchPage {
```

Comment:
It might be worth spreading this very long function signature across at least 2 lines